<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_inclinacao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Inclinação (graus)</string>
    </property>
    <widget class="QLabel" name="label_longitudinal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitudinal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_transversal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Transversal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vertical">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_longitudinal">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_transversal">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_vertical">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_comandos">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Comandos</string>
    </property>
    <widget class="QPushButton" name="pushButton_frente">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cima</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tras">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trás</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_esquerda">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Esquerda</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_direita">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Direita</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_descer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_subir">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_horario">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_antihorario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_subir">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Subir</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_descer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Descer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_antihorario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Colocar imagem</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_horario">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Colocar imagem</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_velocidade">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>70</y>
       <width>20</width>
       <height>131</height>
      </rect>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_velocidade">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>20</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocidade</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_velocidade">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_percent">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>40</y>
       <width>10</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_sensores">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensores (cm)</string>
    </property>
    <widget class="QLabel" name="label_esquerda">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Esquerda</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_esquerda">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_frente">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_frente">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Frente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tras">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_tras">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Trás</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_direita">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_direita">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Direita</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_acima">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_acima">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Acima</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_abaixo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_abaixo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Abaixo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
